# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1468596557 14400
#      Fri Jul 15 11:29:17 2016 -0400
# Node ID 17c59e33b236c45d2e1f77b8dabb6a5fe814eaaa
# Parent  9b832e9caae69736e3f1b3264b036e47b7cca2b8
* oct-conf-post.in.h, mk-octave-config-h.sh: Revert unintended changes from previous changeset.

diff --git a/build-aux/mk-octave-config-h.sh b/build-aux/mk-octave-config-h.sh
--- a/build-aux/mk-octave-config-h.sh
+++ b/build-aux/mk-octave-config-h.sh
@@ -115,17 +115,17 @@ since all of Octave's header files alrea
 #    define OCTAVE_NORETURN
 #    define OCTAVE_UNUSED
 
 /* #    undef HAVE_OCTAVE_DEPRECATED_ATTR */
 /* #    undef HAVE_OCTAVE_NORETURN_ATTR */
 /* #    undef HAVE_OCTAVE_UNUSED_ATTR */
 #  endif
 
-// #  define OCTAVE_USE_DEPRECATED_FUNCTIONS 1
+#  define OCTAVE_USE_DEPRECATED_FUNCTIONS 1
 
 #  if defined (__cplusplus)
 template <typename T>
 static inline void
 octave_unused_parameter (const T&)
 { }
 #  else
 #    define octave_unused_parameter(param) (void) param;
diff --git a/oct-conf-post.in.h b/oct-conf-post.in.h
--- a/oct-conf-post.in.h
+++ b/oct-conf-post.in.h
@@ -45,17 +45,17 @@ along with Octave; see the file COPYING.
 #  define OCTAVE_UNUSED __attribute__ ((__unused__))
 #  define HAVE_OCTAVE_UNUSED_ATTR 1
 #else
 #  define OCTAVE_DEPRECATED(msg)
 #  define OCTAVE_NORETURN
 #  define OCTAVE_UNUSED
 #endif
 
-// #define OCTAVE_USE_DEPRECATED_FUNCTIONS 1
+#define OCTAVE_USE_DEPRECATED_FUNCTIONS 1
 
 #if defined (__cplusplus)
 template <typename T>
 static inline void
 octave_unused_parameter (const T&)
 { }
 #else
 #  define octave_unused_parameter(param) (void) param;
